app/models/case/sar/offender.rb

```diff
@@ -132,4 +131,14 @@ def reassign_gov_uk_dates
   def allow_waiting_for_data_state?
     self.current_state == 'data_to_be_requested'
   end
+
+  private
+
+  def set_subject
+    self.subject = subject_full_name
+  end
+
+  def use_subject_as_requester
+    self.name = self.subject_full_name
+  end
```

Comment:
As above - function name introduces the idea of a 'requester' internally AKA the 'name' field

Reply:
As above, all as existing code :)